pkg/crc/services/dns/dns_darwin.go

```diff
@@ -31,7 +31,9 @@ type resolverFileValues struct {
 func runPostStartForOS(serviceConfig services.ServicePostStartConfig, result *services.ServicePostStartResult) (services.ServicePostStartResult, error) {
 	// Update /etc/hosts file for host
 	if err := updateHostsConfFile(serviceConfig.IP, serviceConfig.BundleMetadata.GetAPIHostname(),
-		serviceConfig.BundleMetadata.GetAppHostname("oauth-openshift")); err != nil {
+		serviceConfig.BundleMetadata.GetAppHostname("oauth-openshift"),
+		serviceConfig.BundleMetadata.GetAppHostname("console-openshift-console"),
+		serviceConfig.BundleMetadata.GetAppHostname("default-route-openshift-image-registry")); err != nil {
```

Comment:
There seems to be a mix of
1) change the existing code to make use of goodhosts-cli to update /etc/hosts
2) add console-openshift-console/default-route-openshift-image-registry to /etc/hosts

I would split this commit along these lines, and merge 1) with the next commit in this series ("Remove `/etc/hosts` file modification on MacOS"). Or replace this with a commit using the same code on linux and macos.

Reply:
This also duplicates the linux code.